Martin Pecha Problem set 1.R

```diff
@@ -0,0 +1,104 @@
+#PROBLEM 1
+#print actual time
+now = Sys.time()
+now
+
+#create a list ”bad list” with 3 positions: ’na’ with values ’NA’ 3 times, ’chars’ with values
+#’a’,’b’,’c’, ’bools’ with values FALSE 2 times and TRUE once
+
+na=c(NA,NA,NA)
+chars=c(a,b,c)
+bools=c(FALSE,FALSE,TRUE)
+
+bad_list=list(na,chars,bools)
+
+#create a data frame ’happyframe’ with similar specification as in previous points: 3 columns
+#’na’, ’chars’, ’bools’; 3 rows
+happyframe=data.frame(na,chars,bools)
+
+#PROBLEM 2
+#write the commands that will calculate the length of numeric vector ’input’ using arithmetic
+#operators and functions only (without using len function)
+x=0
```

Comment:
Nice approach to writing functions. In this function I suggest you create x = 0  within a function and then add values in a for loop.  It will help you to get rid of if-loop.

The way how I see this function is:
- name -> find_len (easier to read and more conventional for programmers)
- delete if loop, instead just start with x = 0
- there might be an issue that, for example vector contains non-numeric values -> you can check that using assertive library 

- all these comments are just the suggestions, so no points taken from that